packages/image_picker/test/image_picker_test.dart

```diff
@@ -98,6 +98,13 @@ void main() {
           throwsArgumentError,
         );
       });
+
+      test('handles a null image path response gracefully', () {
+        channel.setMockMethodCallHandler((MethodCall methodCall) => null);
+
+        ImagePicker.pickImage(source: ImageSource.gallery);
```

Comment:
This test case looks strange. What do you want to test? That the call doesn't throw? Or that it returns null? Either way, it should be clear from the test case.

Reply:
The `File` constructor will throw an error if the image path passed to it is null, which will fail the test. If the test runs, all is good.

I don't know if there's any better way of testing this, but happy to hear if there is. :)

Reply:
I would do something like `expect(await ImagePicker.pickImage(...), isNull);` to state the fact that you expect this to simply return null and nothing else.